rotator_pxi/README.md

```diff
@@ -0,0 +1,60 @@
+# Software Environment of Rotator PXI Controller
+
+*This is the software environment of rotator PXI controller.*
+
+## Overview
+
+*This Dockerfile tries to have the similar software environment in the rotator real-time controller. The NI PXIe-8840 controller is used. The Xenomai dual kernel is installed in the controller. The ipipe, EtherCAT, and HDF5 are installed as well.*
+
+## Difference with the PXI Controller
+
+*1. When installing the Linux kernel, the default configuration (`make defconfig`) is used instead of the customization of real-time contrroler. To do this, the user needs to use `make menuconfig` and enable the [parameters](doc/linuxConfig.md). The user can also copy the existed `.config` file instead.* \
```

Comment:
Markdown knows how to make lists: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Please remove *, see above.

Also there isn't need for \ at the end

the *existed* `.config` -> the *existing* `.config`

Reply:
Fixed in commit: ce7f486. Thanks!